rkt/api_service.go

```diff
@@ -636,8 +639,54 @@ func (s *v1AlphaAPIServer) InspectImage(ctx context.Context, request *v1alpha.In
 	return &v1alpha.InspectImageResponse{Image: image}, nil
 }
 
+type LogsStreamWriter struct {
+	server v1alpha.PublicAPI_GetLogsServer
+}
+
+func (sw LogsStreamWriter) Write(b []byte) (int, error) {
+	if err := sw.server.SendMsg(b); err != nil {
+		return 0, err
+	}
+	return len(b), nil
+}
+
 func (s *v1AlphaAPIServer) GetLogs(request *v1alpha.GetLogsRequest, server v1alpha.PublicAPI_GetLogsServer) error {
-	return fmt.Errorf("not implemented yet")
+	path := fmt.Sprintf("/var/lib/rkt/pods/run/%s/stage1/rootfs/var/log/journal/", request.PodId)
+	jconf := sdjournal.JournalReaderConfig{
+		Path: path,
+	}
+	if request.AppName != "" {
+		jconf.Matches = []sdjournal.Match{
+			{
+				Field: sdjournal.SD_JOURNAL_FIELD_SYSTEMD_UNIT,
+				Value: fmt.Sprintf("%s.service", request.AppName),
```

Comment:
I ponder whether to use [ServiceUnitName()](https://github.com/coreos/rkt/blob/master/stage1/init/common/path.go#L36) just in case we change the way the file name get formed. It is a bit annoying because you'd have to call [types.NewACName](https://github.com/coreos/rkt/blob/master/Godeps/_workspace/src/github.com/appc/spec/schema/types/acname.go#L72)(request.AppName).